Tests/Unit/Stubs/Microsoft365.psm1

```diff
@@ -54126,3 +54126,249 @@ function Set-CsUserCallingSettings
     )
 }
 #endregion
+#region MgDirectoryObject
```

Comment:
We should look at using the beta ones since it needs to be loaded for other resources.

Reply:
So shall I add the dependency from the beta version in the stub util helper?

Reply:
Yes, please. Thanks